src/server/server.js

```diff
@@ -66,7 +65,6 @@ app.get('/', (req, res) => {
  * Serve return_url page
  */
 app.get('/next-step', (req, res) => {
-  // TODO handle sad path cases
```

Comment:
I wasnt sure what this meant. I deleted it so I could create a comment on it

Reply:
Don't even remember anymore what I meant by that 😅 

If it's referring to declined consent and unsupported device error codes, those are handled through the stripe.js iframe events now.

If it's not referring to that, then well... nobody seems to have complained 😆